Makefile

```diff
@@ -403,6 +403,10 @@ svcat:
 	$(MAKE) svcat-for-$(CLIENT_PLATFORM)
 	cp $(BINDIR)/svcat/$(TAG_VERSION)/$(CLIENT_PLATFORM)/$(ARCH)/svcat$(FILE_EXT) $(BINDIR)/svcat/
 
+svcat-install: svcat
+	cp $(BINDIR)/svcat/svcat /usr/local/bin/
```

Comment:
Is this likely to require sudo?

Reply:
🤔 Maybe? It doesn't on my mac but that's not really evidence! heh